# Node Name: handover_controller
handover_controller:

  ros__parameters:

    # Admittance Parameters
    admittance_mass:      [40.0, 40.0, 40.0, 10.0, 10.0, 10.0]
    admittance_damping:   [20.0, 20.0, 20.0, 10.0, 10.0, 10.0]
    # admittance_stiffness: [10.0, 10.0, 10.0, 0.5, 0.5, 0.5]
    admittance_stiffness: [40.0, 40.0, 40.0, 0.0, 0.0, 0.0]

    # FT-Sensor Dead Zone Parameters
    force_dead_zone: 4.0
    torque_dead_zone: 1.0

    # Kinematic Parameters
    maximum_velocity: [1.05, 1.05, 1.57, 1.57, 1.57, 1.57]
    # maximum_acceleration: [0.57, 0.57, 0.57, 0.57, 0.57, 0.57]
    maximum_acceleration: [0.20, 0.20, 0.20, 0.20, 0.20, 0.20]
